playwright-tests/tests/connectWallet.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { test } from '../services/fixture.service';
+import { Tags } from '../test-data/tags.data';
+import { ReefKnotPage } from '../pages/reefKnot.page';
+import { expect } from '@playwright/test';
+import { REEF_KNOT_CONFIG } from '../config';
+import { formatEther } from 'viem';
+import { toCut } from '../services/helpers/helpers';
+import { qase } from 'playwright-qase-reporter';
+
+test.describe.serial(
+  'ReefKnot. Wallet connection',
+  { tag: [Tags.connectedWallet] },
+  async () => {
+    let reefKnotPage: ReefKnotPage;
+
+    test.beforeAll(async ({ reefKnotService }) => {
+      reefKnotPage = new ReefKnotPage(reefKnotService.page);
+      await reefKnotPage.goto();
+      await reefKnotPage.allowUseCookies();
+    });
+
+    test.afterAll(async ({ reefKnotService }) => {
+      await reefKnotService.disconnectWalletForce();
```

Comment:
wdyt to add it into `beforeAll` for pure connection flow?

Reply:
yes, just in case, to be sure of the initial state of the stand before test run